package com.user.Exceptions

class UserNotFoundException() : RuntimeException("No hay un usuario disponible") {
}
